webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -249,7 +214,10 @@ export const ChatWindow: React.FC = () => {
                 </div>
             </div>
             {selectedTab === 'chat' && <ChatRoom />}
-            {selectedTab === 'files' && <ChatResourceList chatId={selectedId} />}
+            {selectedTab === 'documents' && <ChatResourceList />}
+            {selectedTab !== 'chat' && <div className={classes.alerts}>
+                <Alerts />
+            </div>}
```

Comment:
we're adding other tabs soon so this will apply to multiple pages, right?

Reply:
Yes, except the Chat tab where the alert will be above the input box.